user/caribou/module/src/DSO9254AEventConverter.cc

```diff
@@ -0,0 +1,377 @@
+#include "CaribouEvent2StdEventConverter.hh"
+#include "utils/log.hpp"
+#include "CLICTDFrameDecoder.hpp"
+#include "TFile.h"
+#include "TH1D.h"
+#include "time.h"
+
+using namespace eudaq;
+
+namespace{
+  auto dummy0 = eudaq::Factory<eudaq::StdEventConverter>::
+  Register<DSO9254AEvent2StdEventConverter>(DSO9254AEvent2StdEventConverter::m_id_factory);
+}
+
+uint64_t timeConverter( std::string date, std::string time );
```

Comment:
I think this should be a `static` member function of `DSO9254AEvent2StdEventConverter`.